scripts/App.js

```diff
@@ -5,23 +5,24 @@ import Explore from './components/Explore';
 import DocumentTitle from 'react-document-title';
 import { RouteHandler } from 'react-router';
 
-const App = React.createClass({
-  propTypes: {
-    params: PropTypes.object.isRequired,
-    query: PropTypes.object.isRequired
-  },
-
+class App extends React.Component {
   render() {
     return (
       <DocumentTitle title='Sample App'>
         <div className='App'>
-          <Explore />
+          <Explore {...this.props} />
           <hr />
           <RouteHandler {...this.props} />
         </div>
       </DocumentTitle>
     );
   }
-});
+}
+// or declare it in the constructor
+App.propTypes = {
```

Comment:
Since we're running Babel, we can use `static propTypes = { };` inside `App`, it reads better.